src/commands/auth/login.ts

```diff
@@ -0,0 +1,41 @@
+import { Command } from 'commander';
+import { saveApiKey } from '@/src/utils/credentials';
+import { logger } from '@/src/utils/logger';
+import prompts from 'prompts';
+
+export const loginCommand = new Command()
+  .name('login')
+  .description('Set the API key for authentication')
+  .option('-k, --key <key>', 'API key to set')
```

Comment:
Can we make this command simpler? e.g. `phala-cloud-cli auth <api_key>`

Reply:
It is now `phala auth login [api_key]`